.murdock

```diff
@@ -92,6 +92,7 @@ compile() {
     [ ! -d "boards/$board" ] && error "$0: compile: error: board directory \"boards/$board\" doesn't exist"
 
     CCACHE_BASEDIR="$(pwd)" BOARD=$board RIOT_CI_BUILD=1 \
+        PORT=/dev/null \
```

Comment:
as is, this will break murdock workers that actually set ```PORT```.

Reply:
I will remove it as now the check is not script anymore.